keyboards/handwired/dactyl_manuform/4x6_4_3/keymaps/default/keymap.c

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 Adam Tombleson (@rekarnar)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define RAISE MO(_RAISE)
+#define LOWER MO(_LOWER)
+
+enum layer_names {
+    _BASE,
+    _RAISE,
+    _LOWER
+};
+
+enum custom_keycodes {
+    SELECT_LEFT = SAFE_RANGE,
+    SELECT_RIGHT,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case SELECT_RIGHT:
```

Comment:
These can be accomplished with `C(S(KC_RIGHT)` and `C(S(KC_LEFT))` in the keymap directly.  No need for a custom keycode for these.

Reply:
ahh thanks! not sure how i missed those.